cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -87,3 +88,13 @@ func AddCRISocketFlag(flagSet *pflag.FlagSet, criSocket *string) {
 		"Path to the CRI socket to connect. If empty kubeadm will try to auto-detect this value; use this option only if you have more than one CRI installed or if you have non-standard CRI socket.",
 	)
 }
+
+func DefaultInitConfiguration() *kubeadmapiv1beta2.InitConfiguration {
```

Comment:
the function should have an export comment.
the name is way to generic, but we can leave it like that for now.